### DISCUSSION TITLE
Move this repo to the IOOS org

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [ioos/nextgen-dmac#10] [Ideas]
If this repo was moved to the IOOS github org and debranded, it might foster more participation from the community.   It's exciting that RPS Group has funding to accelerate NextGen-DMAC, and it would be great to uses their resources to help build a community effort. 

Having a non-commercial branded org and docs would follows the pattern of successful software packages in the Open Source ecosystem.   For example, the Holoviz suite of visualization packages we use for interactive graphical exploration in the browser are largely developed by Anaconda, Inc, but they all live in the [Holoviz Github org](https://github.com/holoviz), and Anaconda appears nowhere in the documentation of any of the repos.  (The only way to see that Anaconda is mainly responsible is to look at the chief contributors like [Jim Bednar](https://github.com/jbednar), [Philippe Rudiger](https://github.com/philippjfr) and see their affiliation!)

Come to think of it the IOOS sandbox [IOOS Cloud Sandbox](https://github.com/ioos/Cloud-Sandbox), largely developed by RPS Group follows this pattern!    Let's do more of that!  :)

This would make me (and likely others!) feel more like contributing!

**Comment by rsignell-usgs** [comment]
@jonmjoyce @kknee what thinkest ?

**Reply by jonmjoyce** [reply]
I'm deferring to others. Any other Steering Committee members have an opinion on this? I think it'd be good to have participation from IOOS and have more people available for community moderation.

**Comment by emfdavid** [comment]
Certainly would want successful prototypes moved to an IOOS org repository.
There are benefits of early engagement in making the prototype development open source. It also requires careful management of expectations and responsive support for the projects. Maybe this should be a topic for the next steering committee call? It could derail the project if not managed properly.

**Reply by DonaldMoretti** [reply]
Thanks for the feedback and suggestions from both David and Rich, we are certainly open to whatever makes sense and more importantly whatever will help us yield the best participation and results.  Since we included the current links and instructions in Josie's last communication to the Steering Committee, we should probably discuss this topic with that group at the next meeting.  It would be minimal effort to port this to another location, but we should probably consult with that group if even only for awareness and communications.

**Reply by kknee** [reply]
100% agree on moving successful prototypes to the IOOS org. In fact, I think we should establish a process for deciding at what point to move a prototype for wider input and discussion - they certainly don't need to be fully baked before they move, but I'd like to keep discussion of fledgling ideas here so we have a chance to bat them around a bit. 

We established this repo as a mechanism for steering committee communication and collaboration - this is not necessarily about a single package/product/prototype etc, more about giving us a way to interact outside of steering committee calls.

**Comment by dpsnowden** [comment]
I agree with @rsignell-usgs.  Having the entire discussion in a non-commercial space might encourage more contributions.  I think transparency and inclusion of as large a group as possible are important core values.  We hope that there are successful prototypes that result from this project.  Beyond prototypes, I think the lasting contribution from a project like this will be the community that coalesces around the ideas we're discussing.  I'm not suggesting that this can't happen in a corporate organization, just that it's a little more likely in a gov or non-profit space.
